.github/workflows/ci.yml

```diff
@@ -87,8 +87,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jdk: [17]
-        platform: [ubuntu-latest, windows-latest]
+        jdk: [11, 17]
+        platform: [ubuntu-latest] # Removing windows temporarily
```

Comment:
qq, do we know why did we not run integ-tests with 11 before this?

Reply:
We never used them reliably before - notice that `continue-on-error:true` a couple lines down this prevent the tests from surfacing failures until they were enabled on the 2.x line.